spec/services/recommendation_service_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
```

Comment:
Similar to above, let's pair up to iron out the mocks here. Otherwise looks really good!!